### PR TITLE
Revert "Revert "story/BODS-1369: Update guidance and .txt for field name descriptions""

### PR DESCRIPTION
**Pull request by ndemenezes**
Adds updated guidance text back

### CODE REVIEW

File: transit_odp/timetables/tests/test_csv.py [hunk 1]
```diff
@@ -42,18 +42,16 @@ def test_service_in_bods_but_not_in_otc():
         assert row["Published Status"] == "Published"
         assert row["OTC Status"] == "Unregistered"
         assert row["Scope Status"] == "In Scope"
-        assert row["DQ Score"] == "33%"
```
Comment (alvak1) [inline transit_odp/timetables/tests/test_csv.py:45]: Not on this line but there are a few test cases that are failing

**Review by alvak1 — COMMENTED** [review]
comment added

File: transit_odp/timetables/tests/test_csv.py [hunk 2]
```diff
@@ -42,18 +42,16 @@ def test_service_in_bods_but_not_in_otc():
...
-        assert row["BODS Compliant"] == "NO"
-        assert row["XML Filename"] == txc_file_attributes.filename
-        assert row["Last Updated Date"] == dataset.live_revision.published_at
+        assert row["XML:Filename"] == txc_file_attributes.filename
```
Comment (alvak1) [inline transit_odp/timetables/tests/test_csv.py:45]: Add a test case to verify the order of the columns

**Review by alvak1 — COMMENTED** [review]
comment added

File: transit_odp/timetables/tests/test_csv.py (hunk 2 above)
Comment (ndemenezes) [reply transit_odp/timetables/tests/test_csv.py:105]: This has been done in: `test_csv_output_order`

File: transit_odp/timetables/tests/test_csv.py (hunk 1 above)
Comment (ndemenezes) [reply transit_odp/timetables/tests/test_csv.py:45]: Fixed this @alvak1

**Review by alvak1 — APPROVED** [review]
Approved

**Pull request merged**